requirements.txt

```diff
@@ -9,3 +9,4 @@ python-neutronclient
 keystoneauth1
 boto3
 python-novaclient
+python-swiftclient

```

Comment:
We should refactor this file for including specific range of version e.g: python-swiftclient >= 3.0.0